metecho/api/serializers.py

```diff
@@ -443,28 +452,40 @@ def get_branch_url(self, obj) -> Optional[str]:
         return None
 
     def get_branch_diff_url(self, obj) -> Optional[str]:
-        epic = obj.epic
-        epic_branch = epic.branch_name
-        project = epic.project
+        base_branch = obj.get_base()
+        project = obj.root_project
         repo_owner = project.repo_owner
         repo_name = project.repo_name
         branch = obj.branch_name
-        if repo_owner and repo_name and epic_branch and branch:
+        if repo_owner and repo_name and base_branch and branch:
             return (
                 f"https://github.com/{repo_owner}/{repo_name}/compare/"
-                f"{epic_branch}...{branch}"
+                f"{base_branch}...{branch}"
```

Comment:
Tasks attached to projects will be compared against the Project branch (not a particular Epic branch)

Reply:
That sounds correct 👍